src/storage/mod.rs

```diff
@@ -247,7 +247,12 @@ impl<E: Engine, L: LockMgr> Storage<E, L> {
 
             // The bypass_locks set will be checked at most once. `TsSet::vec` is more efficient
             // here.
-            let bypass_locks = TsSet::vec(ctx.take_resolved_locks());
+            let bypass_locks = TsSet::vec(
+                ctx.take_resolved_locks()
+                    .into_iter()
+                    .map(Into::into)
+                    .collect(),
+            );
```

Comment:
Why not implement the type conversion in `TsSet::new()`?

Reply:
I want to avoid implicitly converting from u64 to TimeStamp. It should be an explicit operation to do that. I can factor the conversion out into a function in `Storage` though.